### PR TITLE
Corrects a couple items

### LINKED ISSUE #5420: Need update item 16033, 21011, 28101
* **rAthena Hash**: latest

* **Client Date**: not related

* **Server Mode**: renewal

* **Description of Issue**: need update item according to item info divine pride item id 16033, 21011, 28101
  * Result: 
  * Expected Result: 
  * How to Reproduce: 
  * Official Information:
  

* **Modifications that may affect results**:

### PR DESCRIPTION
**Pull request by aleos89**
* **Addressed Issue(s)**: Fixes #5420

* **Server Mode**: Renewal

* **Description of Pull Request**: 
  * Fixes Robot's Arm, Gigantic Blade, and Tornado Axe buy/sell price, range, and job equip.
Thanks to @mazvi!

### CODE REVIEW

File: db/re/item_db.txt [hunk 1]
```diff
@@ -8959,7 +8959,7 @@
 16030,Pilebuncker_S,Pile Bunker S,5,20,,3000,400,,1,1,0x00000400,56,2,2,4,130,1,8,{ bonus bAspdRate,getrefine()/2; },{},{}
 16031,Pilebuncker_P,Pile Bunker P,5,20,,4000,450,,1,,0x00000400,56,2,2,4,130,1,8,{ bonus bBaseAtk,getrefine()*5; },{},{}
 16032,Pilebuncker_T,Pile Bunker T,5,20,,3500,400,,1,1,0x00000400,56,2,2,4,130,1,8,{ bonus bUseSPrate,getrefine()*-1; },{},{}
-16033,Robot's_Arm,Robot's Mechanical Arm,5,20,,3000,195,,1,2,0x00000400,56,2,2,4,130,1,8,{ bonus bUnbreakableWeapon; bonus bCritical,20; },{},{}
+16033,Robot's_Arm,Robot's Arm,5,30000,,3000,195,,5,2,0x0004C5B2,56,2,2,4,130,1,8,{ bonus bUnbreakableWeapon; bonus bCritical,20; },{},{}
```
Comment (Atemo) [inline db/re/item_db.txt:8962]: The range were fine according to aegis data
```suggestion
16033,Robot's_Arm,Robot's Arm,5,30000,,3000,195,,1,2,0x0004C5B2,56,2,2,4,130,1,8,{ bonus bUnbreakableWeapon; bonus bCritical,20; },{},{}
```

File: db/re/item_db.txt [hunk 2]
```diff
@@ -11292,7 +11292,7 @@
 21008,Small_Karasuma,Small Karasuma,5,20,,1000,170,,1,2,0x00004082,63,2,34,4,50,1,3,{ bonus bCritAtkRate,getrefine()*2; bonus3 bAutoSpell,"NPC_CRITICALWOUND",1,50; },{},{}
 21009,Thanos_Great_Sword,Thanos Great Sword,5,10,,1300,280:120,,1,1,0x00004082,56,2,34,4,120,1,3,{ bonus bInt,6; bonus bVit,6; bonus bLuk,-6; bonus2 bHPDrainRate,50,5; bonus2 bSPDrainRate,10,5; bonus2 bHPLossRate,100,10000; },{},{ heal -1000,0; }
 21010,Tw_Sword_Of_Evil_Slayer,Evil Slayer Slaher Sword,5,0,,2000,225,,1,1,0x00004082,63,2,34,3,100,1,3,{ bonus2 bAddRace,RC_Demon,10; bonus2 bAddRace,RC_Undead,10; .@r = getrefine(); if(.@r>=9) { .@dmg = 5; if(.@r>=12) { .@dmg += 7; } bonus2 bAddClass,Class_All,.@dmg; } },{},{}
-21011,Giant_Blade,Gigantic Blade,5,10,,5000,300,,1,1,0x00004082,56,2,34,4,130,1,3,{ bonus bUnbreakableWeapon; bonus bAspdRate,-5; bonus bCritAtkRate,getrefine(); if(readparam(bStr)<=110){ bonus bAtk,-250; } },{},{}
+21011,Giant_Blade,Gigantic Blade,5,30000,,5000,300,,5,1,0x00004082,56,2,34,4,130,1,3,{ bonus bUnbreakableWeapon; bonus bAspdRate,-5; bonus bCritAtkRate,getrefine(); if(readparam(bStr)<=110){ bonus bAtk,-250; } },{},{}
```
Comment (Atemo) [inline db/re/item_db.txt:11295]: The range were fine according to aegis data
```suggestion
21011,Giant_Blade,Gigantic Blade,5,30000,,5000,300,,1,1,0x00004082,56,2,34,4,130,1,3,{ bonus bUnbreakableWeapon; bonus bAspdRate,-5; bonus bCritAtkRate,getrefine(); if(readparam(bStr)<=110){ bonus bAtk,-250; } },{},{}
```

File: db/re/item_db.txt [hunk 3]
```diff
@@ -12550,7 +12550,7 @@
 // New Axes
 //===================================================================
 28100,Thanos_Axe,Thanos Axe,5,10,,4000,300:80,,1,1,0x000444A2,56,2,34,4,120,1,7,{ bonus bInt,6; bonus bVit,6; bonus bLuk,-6; bonus2 bHPDrainRate,50,5; bonus2 bSPDrainRate,10,5; bonus2 bHPLossRate,100,10000; },{},{ heal -1000,0; }
-28101,Tornado_Axe,Axe Tornado,5,10,,4000,320,,,1,0x00000400,56,2,34,4,100,1,7,{ bonus bAtkEle,Ele_Wind; bonus2 bSkillCooldown,"NC_AXETORNADO",-1000; },{},{}
+28101,Tornado_Axe,Tornado Axe,5,30000,,4000,320,,5,1,0x000444A2,56,2,34,4,100,1,7,{ bonus bAtkEle,Ele_Wind; bonus2 bSkillCooldown,"NC_AXETORNADO",-1000; },{},{}
```
Comment (Atemo) [inline db/re/item_db.txt:12553]:
```suggestion
28101,Tornado_Axe,Tornado Axe,5,30000,,4000,320,,1,1,0x000444A2,56,2,34,4,100,1,7,{ bonus bAtkEle,Ele_Wind; bonus2 bSkillCooldown,"NC_AXETORNADO",-1000; },{},{}
```

File: sql-files/item_db_re.sql [hunk 4]
```diff
@@ -8991,7 +8991,7 @@ REPLACE INTO `item_db_re` VALUES (16029,'Noble_Cross','Noble Cross',5,10,NULL,15
 REPLACE INTO `item_db_re` VALUES (16030,'Pilebuncker_S','Pile Bunker S',5,20,NULL,3000,'400',NULL,1,1,0x00000400,56,2,2,4,'130',1,8,'bonus bAspdRate,getrefine()/2;',NULL,NULL);
 REPLACE INTO `item_db_re` VALUES (16031,'Pilebuncker_P','Pile Bunker P',5,20,NULL,4000,'450',NULL,1,NULL,0x00000400,56,2,2,4,'130',1,8,'bonus bBaseAtk,getrefine()*5;',NULL,NULL);
 REPLACE INTO `item_db_re` VALUES (16032,'Pilebuncker_T','Pile Bunker T',5,20,NULL,3500,'400',NULL,1,1,0x00000400,56,2,2,4,'130',1,8,'bonus bUseSPrate,getrefine()*-1;',NULL,NULL);
-REPLACE INTO `item_db_re` VALUES (16033,'Robot\'s_Arm','Robot\'s Mechanical Arm',5,20,NULL,3000,'195',NULL,1,2,0x00000400,56,2,2,4,'130',1,8,'bonus bUnbreakableWeapon; bonus bCritical,20;',NULL,NULL);
+REPLACE INTO `item_db_re` VALUES (16033,'Robot\'s_Arm','Robot\'s Arm',5,30000,NULL,3000,'195',NULL,5,2,0x0004C5B2,56,2,2,4,'130',1,8,'bonus bUnbreakableWeapon; bonus bCritical,20;',NULL,NULL);
```
Comment (Atemo) [inline sql-files/item_db_re.sql:8994]:
```suggestion
REPLACE INTO `item_db_re` VALUES (16033,'Robot\'s_Arm','Robot\'s Arm',5,30000,NULL,3000,'195',NULL,1,2,0x0004C5B2,56,2,2,4,'130',1,8,'bonus bUnbreakableWeapon; bonus bCritical,20;',NULL,NULL);
```

File: sql-files/item_db_re.sql [hunk 5]
```diff
@@ -11324,7 +11324,7 @@ REPLACE INTO `item_db_re` VALUES (21007,'Heavy_Sword','Heavy Greatsword',5,20,NU
...
 REPLACE INTO `item_db_re` VALUES (21009,'Thanos_Great_Sword','Thanos Great Sword',5,10,NULL,1300,'280:120',NULL,1,1,0x00004082,56,2,34,4,'120',1,3,'bonus bInt,6; bonus bVit,6; bonus bLuk,-6; bonus2 bHPDrainRate,50,5; bonus2 bSPDrainRate,10,5; bonus2 bHPLossRate,100,10000;',NULL,'heal -1000,0;');
 REPLACE INTO `item_db_re` VALUES (21010,'Tw_Sword_Of_Evil_Slayer','Evil Slayer Slaher Sword',5,0,NULL,2000,'225',NULL,1,1,0x00004082,63,2,34,3,'100',1,3,'bonus2 bAddRace,RC_Demon,10; bonus2 bAddRace,RC_Undead,10; .@r = getrefine(); if(.@r>=9) { .@dmg = 5; if(.@r>=12) { .@dmg += 7; } bonus2 bAddClass,Class_All,.@dmg; }',NULL,NULL);
-REPLACE INTO `item_db_re` VALUES (21011,'Giant_Blade','Gigantic Blade',5,10,NULL,5000,'300',NULL,1,1,0x00004082,56,2,34,4,'130',1,3,'bonus bUnbreakableWeapon; bonus bAspdRate,-5; bonus bCritAtkRate,getrefine(); if(readparam(bStr)<=110){ bonus bAtk,-250; }',NULL,NULL);
+REPLACE INTO `item_db_re` VALUES (21011,'Giant_Blade','Gigantic Blade',5,30000,NULL,5000,'300',NULL,5,1,0x00004082,56,2,34,4,'130',1,3,'bonus bUnbreakableWeapon; bonus bAspdRate,-5; bonus bCritAtkRate,getrefine(); if(readparam(bStr)<=110){ bonus bAtk,-250; }',NULL,NULL);
```
Comment (Atemo) [inline sql-files/item_db_re.sql:11327]:
```suggestion
REPLACE INTO `item_db_re` VALUES (21011,'Giant_Blade','Gigantic Blade',5,30000,NULL,5000,'300',NULL,1,1,0x00004082,56,2,34,4,'130',1,3,'bonus bUnbreakableWeapon; bonus bAspdRate,-5; bonus bCritAtkRate,getrefine(); if(readparam(bStr)<=110){ bonus bAtk,-250; }',NULL,NULL);
```

File: sql-files/item_db_re.sql [hunk 6]
```diff
@@ -12582,7 +12582,7 @@ REPLACE INTO `item_db_re` VALUES (28044,'Agudo_Filo','Agudo Filo',5,20,NULL,2000
 # New Axes
 #===================================================================
 REPLACE INTO `item_db_re` VALUES (28100,'Thanos_Axe','Thanos Axe',5,10,NULL,4000,'300:80',NULL,1,1,0x000444A2,56,2,34,4,'120',1,7,'bonus bInt,6; bonus bVit,6; bonus bLuk,-6; bonus2 bHPDrainRate,50,5; bonus2 bSPDrainRate,10,5; bonus2 bHPLossRate,100,10000;',NULL,'heal -1000,0;');
-REPLACE INTO `item_db_re` VALUES (28101,'Tornado_Axe','Axe Tornado',5,10,NULL,4000,'320',NULL,NULL,1,0x00000400,56,2,34,4,'100',1,7,'bonus bAtkEle,Ele_Wind; bonus2 bSkillCooldown,"NC_AXETORNADO",-1000;',NULL,NULL);
+REPLACE INTO `item_db_re` VALUES (28101,'Tornado_Axe','Tornado Axe',5,30000,NULL,4000,'320',NULL,5,1,0x000444A2,56,2,34,4,'100',1,7,'bonus bAtkEle,Ele_Wind; bonus2 bSkillCooldown,"NC_AXETORNADO",-1000;',NULL,NULL);
```
Comment (Atemo) [inline sql-files/item_db_re.sql:12585]:
```suggestion
REPLACE INTO `item_db_re` VALUES (28101,'Tornado_Axe','Tornado Axe',5,30000,NULL,4000,'320',NULL,1,1,0x000444A2,56,2,34,4,'100',1,7,'bonus bAtkEle,Ele_Wind; bonus2 bSkillCooldown,"NC_AXETORNADO",-1000;',NULL,NULL);
```

**Review by Atemo — CHANGES_REQUESTED** [review]

**Review by Lemongrass3110 — APPROVED** [review]

**Review by Atemo — APPROVED** [review]

**Pull request merged**